test/unit/custom_index_test.lua

```diff
@@ -198,7 +198,7 @@ function g.test_tree_index_multikey(cg)
 end
 
 function g.test_memtx_tree_functional_index_broken(cg)
-    t.skip_if(_TARANTOOL < "2" or not helpers.memtx_func_index_is_broken(),
+    t.skip_if(_TARANTOOL < "2" or helpers.memtx_func_index_is_supported(),
               "Unsupported Tarantool version")
```

Comment:
It seems `_TARANTOOL < "2"` is already part of the `memtx_func_index_is_supported()` check.

Reply:
Or I don't understand some logic here. We just check that expirationd raises a meaningful error and don't touch the space, corrent?

Reply:
> It seems `_TARANTOOL < "2"` is already part of the `memtx_func_index_is_supported()` check.

But Tarantool does not and we don't create a functional index in a test helper. I've split the condition into a two to make it clear.

Reply:
> Or I don't understand some logic here. We just check that expirationd raises a meaningful error and don't touch the space, corrent?

Yes, it is.

Reply:
> I've split the condition into a two to make it clear.

Nice idea, thanks.